doc/discretization.rst

```diff
@@ -0,0 +1,282 @@
+$$
+\mathbf{V}^k_h &= \left\lbrace \mathbf{v} \in L^2(\Omega_h)^N \text{ such that }
+\mathbf{v}|_E \in \lbrack P^k(E) \rbrack^N, \text{ for all } E \in \Omega_h
+\right\rbrace, \\
+\mathbf{W}^k_h &= \left\lbrace \mathbf{w} \in L^2(\Omega_h)^{N\times d} \text{ such that }
+\mathbf{w}|_E \in \lbrack P^k(E) \rbrack^{N\times d}, \text{ for all } E \in \Omega_h
+\right\rbrace,
+$$
+where $N = d + 2 + N_s$, $d$ is the spatial dimension, and $N_s$ is the total number of
+mixture species. Here, $P^k(E)$ denotes a polynomial space on $E$ consisting of functions
+of degree $\leq k$. The DG formulation is obtained by multiplying by ''test functions''
+$\mathbf{v}_h \in \mathbf{V}^k_h$, $\mathbf{w}_h \in \mathbf{W}^k_h$
+(one for each equation repsectively) and integrating over each element.
+The resulting DG problem reads as follows. Find $(\mathbf{Q}_h,
+\mathbf{\Sigma}_h) \in \mathbf{V}^k_h \times \mathbf{W}^k_h$ such that, for all
+$(\mathbf{v}_h, \mathbf{w}_h) \in \mathbf{V}^k_h \times \mathbf{W}^k_h$, we have:
+
+$$
+\sum_{E\in\Omega_h} \left\lbrack \int_E \mathbf{v}_h\cdot\frac{\partial \mathbf{Q}_h}
+{\partial t} d\Omega + \oint_{\partial E} \mathbf{v}_h\mathbf{h} d\sigma - \int_E \nabla
+\mathbf{v}_h\cdot\mathbf{F}(\mathbf{Q}_h, \mathbf{\Sigma}_h)d\Omega\right\rbrack &=
+\sum_{E\in\Omega_h} \int_E \mathbf{v}_h\cdot\mathbf{S}_h d\Omega, \\
+\sum_{E\in\Omega_h}\left\lbrack \int_E \mathbf{w}_h\cdot \mathbf{\Sigma}_h d\Omega -
+\oint_{\partial E} \mathbf{w}_h\cdot\mathbf{H}_s d\sigma + \int_E \nabla\mathbf{w}_h\cdot
+\mathbf{Q}_h d\Omega\right\rbrack &= 0,
+$$
+
+where $\mathbf{h} = \mathbf{h}_e - \mathbf{h}_v$ is a *numerical flux* consisting of a
+*invicid* and *viscous* component respectively, and $\mathbf{H}_s$ is the
+*gradient numerical flux* for the auxiliary equation. Here, we use the subscript "h" to
+denote discretized quantities.
+
+Since $\mathbf{F}^I(\mathbf{Q}_h)\cdot\mathbf{n}$ is discontinuous, the quantities are
+allowed to vary on either side of a shared element boundary. That is, $\mathbf{F}^I
+(\mathbf{Q}^+_h)\cdot\mathbf{n}^+ \neq \mathbf{F}^I(\mathbf{Q}^-_h)\cdot\mathbf{n}^-$,
+where $\mathbf{n}^\pm$ denotes the ''cell-local'' (or interior) normal with respect to the
+element $E^\pm$ which share a face: $\partial E^+ \cap \partial E^- \neq \emptyset$.
+Similarly for $\mathbf{F}^V(\mathbf{Q}_h, \mathbf{\Sigma}_h)\cdot\mathbf{n}$ and
+$\mathbf{Q}_h\mathbf{n}$.
+
+Expanding out the trial and test functions (component-wise) in terms of the local
+element basis in each element:
+$$
+\mathbf{Q}(\mathbf{x}, t)_h|_E = \sum_{i=1}^n \mathbf{Q}_i(t)\phi_i^k(\mathbf{x}), \quad
+\mathbf{\Sigma}(\mathbf{x})_h|_E = \sum_{i=1}^n \mathbf{\Sigma}_i\phi_i^k(\mathbf{x}), \\
+\mathbf{v}(\mathbf{x})_h|_E = \sum_{i=1}^n \mathbf{v}_i\phi_i^k(\mathbf{x}), \quad
+\mathbf{w}(\mathbf{x})_h|_E = \sum_{i=1}^n \mathbf{w}_i\phi_i^k(\mathbf{x}),
+$$
+allows us to obtain a set of algebraic equations for the prognostic state $\mathbf{Q}_h$ and
```

Comment:
Is "prognostic" a word that's used outside of weather forecasting? Never heard it before @thomasgibson showed up. :)

Reply:
Yes. It's a word you never want to hear your M.D. say.

Reply:
I removed the medical terminology in https://github.com/illinois-ceesd/mirgecom/pull/345/commits/8554dde3589abe8e3f97e437f0c178764e7b30a9 :)